WeakAuras/GenericTrigger.lua

```diff
@@ -1337,29 +1330,18 @@ function GenericTrigger.Add(data, region)
                     hasParam = true
                   end
                 elseif trueEvent:match("^UNIT_") then
-                  local includePets
-                  if i == "grouppets" then
+                  isUnitEvent = true
+
+                  if string.lower(strsub(i, #i - 3)) == "pets" then
+                    i = strsub(i, 1, #i-4)
                     includePets = "PlayersAndPets"
-                    i = "group"
-                  elseif i == "grouppetsonly" then
+                  elseif string.lower(strsub(i, #i - 7)) == "petsonly" then
                     includePets = "PetsOnly"
-                    i = "group"
-                  end
-                  MultiUnitLoop(
```

Comment:
It turns out that this expanding of multi units is not necessary at all. Since it will be done later by LoadDisplay/LoadEvent